contributing/CDIP explainer.md

```diff
@@ -0,0 +1,83 @@
+# CDIP Explainer
+
+## What is a CDIP
+
+CDIP stands for Community Development Initiative Proposal. The CDIP is the scoping and framing tool for for anyone interested in contributing to the MakerDAO community initiatives or resources. CDIPs are used to propose new initiatives or expand the scope of [existing initiatives and resources](https://community-development.makerdao.com/). A proposal should combine the contributor’s ideas, skills, and abilities with our shared goal of improving the resources available to the MakerDAO community.
+
+After starting a CDIP, the owner is responsible for sharing it with the community for feedback. There is a dedicated Guide that is assigned to help a CDIP Owner through the approval process. Once approved, work can be started on the new or expanded initiative.
+
+### What are the benefits of a CDIP
+
+The CDIP provides a standard contributing template for Maker’s community. The format provides a few benefits: collect and organize relevant information in a standardized format to make ideas easier to understand, add an efficient and transparent evaluation layer to contribution proposals, simplify the ability to archive ideas and search details about them later, and ultimately serve as a tool to streamline how we review and approve work as a community. CDIPs scale the ability to execute the ideas of the community.
+
+### How is a CDIP started
+
+Use this [CDIP template doc](http://bit.ly/Comm_Dev_CDIP) to start the process. To submit a CDIP, simply make a copy of the template, fill out the first page (step 1), and post a link to the doc in our group #[Community Development](https://chat.makerdao.com/channel/community-development) rocketchat. A Guide will reach out to help the CDIP owner through the next steps.
```

Comment:
Lets avoid the use of url shorteners. This also links to a personal account. 
lets not use the name 'rocketchat/, please just call it our chat.